micropub.php

```diff
@@ -89,11 +122,16 @@ public static function parse_query($wp) {
       $args['post_author'] = $user_id;
     }
 
-    if (!isset($_POST['url']) || $_POST['action'] == 'create') {
+    if (!isset($_POST['u']) || $_POST['action'] == 'create') {
```

Comment:
keep `url` also for backward compatibility?

also we need to add `u` to line 107.